llm/gguf.go

```diff
@@ -224,33 +233,6 @@ func (llm *gguf) Decode(rs io.ReadSeeker) error {
 
 	// patch KV with parameter count
 	llm.kv["general.parameter_count"] = llm.parameters
-
-	alignment, ok := llm.kv["general.alignment"].(uint32)
-	if !ok {
-		alignment = 32
-	}
-
-	offset, err := rs.Seek(0, io.SeekCurrent)
-	if err != nil {
-		return err
-	}
-
-	padding := llm.padding(offset, int64(alignment))
-	if _, err := rs.Seek(padding, io.SeekCurrent); err != nil {
-		return err
-	}
-
-	for _, tensor := range llm.tensors {
```

Comment:
this should be kept to validate the file isn't truncated (or too long) though you can probably skip to the last tensor instead of reading each tensor individually

Reply:
Right. Bringing it back.